mmv1/products/beyondcorp/terraform.yaml

```diff
@@ -47,6 +47,31 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         vars:
           app_gateway_name: "my-app-gateway"
           display_name: "some display name"
+  AppConnection: !ruby/object:Overrides::Terraform::ResourceOverride
+    autogen_async: true
+    properties:
+      gateway: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "beyondcorp_app_connection_basic"
+        primary_resource_id: "app_connection"
+        primary_resource_name: "fmt.Sprintf(\"tf_test_my_app_connection%s\", context[\"random_suffix\"])"
+        vars:
+          account_id: "my-account"
+          app_connector_name: "my-app-connector"
+          app_connection_name: "my-app-connection"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "beyondcorp_app_connection_full"
+        # skip the test until gateway delete issue is resolved
+        skip_test: true
```

Comment:
Could I receive more context on this test being disabled. We aren't able to test some functionality without it.

Reply:
This test should run now.